components/onboarding/internal/adapters/postgres/ledger/ledger.go

```
@@ -34,6 +34,12 @@ func (t *LedgerPostgreSQLModel) ToEntity() *mmodel.Ledger {
 		Description: t.StatusDescription,
 	}
 
+	var settings *mmodel.LedgerSettings
+	if len(t.Settings) > 0 {
```

golangci-lint (2.11.3)

[error] missing whitespace above this line (no shared variables above if)

(wsl_v5)